src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/Generated/IContainerServiceOperations.cs

```diff
@@ -1,172 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@brendandburns I'm assuming this file was deleted for `IContainerServicesOperations.cs`, but that file does not exist